Template/deviceConfiguration.json

```diff
@@ -37,7 +46,7 @@
 					"LoRaWanNetworkSrvModule": {
 						"type": "docker",
 						"settings": {
-							"image": "loraedge/lorawannetworksrvmodule:0.3.0-preview-arm32v7",
+							"image": "loraedge/lorawannetworksrvmodule:0.4.0-preview-arm32v7",
 							"createOptions": "{\"ExposedPorts\": { \"1680/udp\": {}}, \"HostConfig\": {  \"PortBindings\": {\"1680/udp\": [  { \"HostPort\": \"1680\", \"HostIp\":\"172.17.0.1\" } ]}}}"
```

Comment:
@Mandur these one should be agnostic right?

Reply:
Yes correct  0.4.0-preview should be name for both

Reply:
Updated!

Reply:
Same applies to pktfwd module

Reply:
Yes, both images are now ready